package.json

```
@@ -0,0 +1,51 @@
+{
+  "name": "payment-sdk",
+  "author": "",
+  "license": "ISC",
+  "version": "1.0.0",
+  "description": "",
+  "homepage": "https://github.com/XcooBee/payment-sdk-js#readme",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/XcooBee/payment-sdk-js.git"
+  },
+  "bugs": {
+    "url": "https://github.com/XcooBee/payment-sdk-js/issues"
+  },
+  "main": "dist/node/node.js",
+  "module": "dist/node/node.js",
+  "browser": "dist/browser/browser.js",
+  "scripts": {
+    "test": "mocha -r ts-node/register tests/**/*.test.ts",
+    "coverage": "nyc -r lcov -e .ts -x \"*.test.ts\" npm run test",
+    "docs": "typedoc",
+    "start": "tsc && tsc -p tsconfig.browser.json"
```

Let's rename this to `build` (or `compile`).
```suggestion
    "build": "tsc && tsc -p tsconfig.browser.json"
```